ansible_base/authentication/utils/claims.py

```diff
@@ -543,8 +543,11 @@ def _give_permission(self, role_definition: CommonModel, obj: Union[AbstractOrga
     def _remove_permission(self, role_definition: CommonModel, obj: Union[AbstractOrganization, AbstractTeam, None] = None) -> None:
         if obj:
             logger.info(
-                _("Removing role '{rd}' from user '{username}' in '{object}").format(
-                    rd=role_definition.name, username=self.user.username, object=obj.__class__.__name__
+                _("Removing role '{rd}' from user '{username}' on '{object}' id {id}").format(
```

Comment:
Note, I found a missing `'` in the log and also thought I'd add the id (if its available) while I was in here changing things.